core/src/main/scala/io/finch/internal/ToResponse.scala

```diff
@@ -44,13 +44,26 @@ trait LowPriorityToResponseInstances {
 
 object ToResponse extends LowPriorityToResponseInstances {
 
+  private val newLineDelimeter = Buf.Utf8("\n")
+
   implicit def asyncBufToResponse[CT <: String](implicit
     w: Witness.Aux[CT]
-  ): Aux[AsyncStream[Buf], CT] = instance { a =>
+  ): Aux[AsyncStream[Buf], CT] = asyncStreamResponseBuilder(identity)
+
+  implicit def asyncStreamToResponse[A, CT <: String](implicit
```

Comment:
Shouldn't we fix content type to JSON here given that this particular encoding only makes sense for JSON objects?